### PR TITLE
add details when create new web app

### PR DESCRIPTION
**Pull request by Windsooon**
I'm not sure the link of the screenshot.

### CODE REVIEW

**Comment by hjwp** [comment]
Hi there, thanks very much for the PR!  we always *really* appreciate anyone trying to help improve our docs.

With that said, we do mention that the user has to use "manual configuration" twice in the document already, once in the overview and once in the detail section.  what made you want to add this third warning again?

maybe just saying soewhere, explicitly, "don't choose the django option, that's for new projects only?"

**Comment by Windsooon** [comment]
I like pythonanywhere and you guys are so nice. I will try to help improve the docs. How about this? I think to add some screenshot will more friendly to users.

File: articles/DeployExistingDjangoProject.md [hunk 1]
```diff
@@ -28,7 +31,7 @@ Here's an overview of the steps involved.
 
 1. Upload your code to PythonAnywhere
 2. Set up a virtualenv and install Django and any other requirements
-3. Set up your web app using manual config and your WSGI file.
+3. Set up your web app using **manual config** (choose **Django** if you want to create a new project) and your WSGI file.
```
Comment (hjwp) [inline articles/DeployExistingDjangoProject.md]: I would change this do "do not choose the *Django* option, that's for new projects only"

File: articles/DeployExistingDjangoProject.md [hunk 2]
```diff
@@ -45,7 +48,25 @@ $ git clone https://github.com/myusername/myproject.git
 That's the solution we recommend, but there are a few different methods
 documented on the [Uploading code](/pages/FTP) page,
 
+For example, If your directory structure look like this
 
+```bash
```
Comment (hjwp) [inline articles/DeployExistingDjangoProject.md]: there doesn't seem to be any point to this section atm?  let's remove for now?

File: articles/DeployExistingDjangoProject.md [hunk 3]
```diff
@@ -83,10 +104,26 @@ create your virtualenv).
 ### Enter your virtualenv name
 
 Once that's done, **enter the name of your virtualenv** in the Virtualenv
-section on the web tab and click OK.  You can just use its short name
+section on the web tab and click OK.
+
+![virtualenv_web_app](path_to_the_screenshot)
```
Comment (hjwp) [inline articles/DeployExistingDjangoProject.md]: good idea!  happy to accept screenshots, you can add them to the PR, put them in the **files** folder.  there's an example in *Education.md*

File: articles/DeployExistingDjangoProject.md [hunk 4]
```diff
@@ -83,10 +104,26 @@ create your virtualenv).
...
+
+You can just use its short name
 "mysite-virtualenv", and it will automatically complete to its full path in
 /home/username/.virtualenvs.
 
+### Enter your code name
```
Comment (hjwp) [inline articles/DeployExistingDjangoProject.md]: this is really optional at the moment.  i would either remove this section altogether (to keep things as simple as possible).   if you really, really, really want to keep it, at least change the section title to: Optional: Enter path to source code

File: articles/DeployExistingDjangoProject.md (hunk 3 above)
Comment (hjwp) [inline articles/DeployExistingDjangoProject.md]: oh I'm sorry I see you already have those!  thanks so much :)

**Comment by Windsooon** [comment]
Feel free to edit as you like.

**Comment by hjwp** [comment]
awesome.  that's live now.  you know what would make it even better?  if those screenshots had some example entries in them, eg in the virtualenv path and the static files  menu...

**Pull request merged**